examples/py/async-binance-create-option-order.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import os
+import sys
+from pprint import pprint
+
+root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(root + '/python')
+
+import ccxt.async_support as ccxt  # noqa: E402
+
+
```

Comment:
We could probably add a comment to each file saying something like this:

```
This example uses the implicit API, so this is not the most ergonomic way to handle it, in the future we will have options properly unified which will make things much easier.
You can check if they're already ready-to-use with the unified methods  (createOrder, fetchOrder, fetchTicker, etc) by doing `is_unified = exchange.has['option']`
```

wdyt?

Reply:
Yeah I think that's a good idea, I'll add a comment to each file